Polyhedron/demo/Polyhedron/Plugins/Point_set/Surface_reconstruction_plugin.cpp

```diff
@@ -889,13 +984,26 @@ void Polyhedron_demo_surface_reconstruction_plugin::automatic_reconstruction
 	      std::cerr << "Advancing front reconstruction... ";
 	      time.restart();
 
-	      Scene_polyhedron_item* reco_item = new Scene_polyhedron_item(Polyhedron());
-	      SurfaceReconstruction::advancing_front (*points, reco_item, 10. * (std::max)(noise_size, aniso_size));
-	      
-	      reco_item->setName(tr("%1 (advancing front)").arg(scene->item(index)->name()));
-	      reco_item->setColor(Qt::lightGray);
-	      reco_item->setRenderingMode(FlatPlusEdges);
-	      scene->addItem(reco_item);
+              if(mw->property("is_polyhedron_mode").toBool())
```

Comment:
In the Edition menu, preferences, you can chose a default type of facegraph items.